test/qa/test_meshnet_routing.py

```diff
@@ -86,3 +91,173 @@ def test_killswitch_exitnode(lan_discovery: bool, local: bool):
     sh.nordvpn.disconnect()
     assert not daemon.is_connected()
     assert network.is_available()
+
+
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(60)
+def test_route_to_peer_when_already_routing():
```

Comment:
is it while `connecting` or is already connected?

Reply:
It is when already connected :smile: 
I cannot think of better test name for this one. Maybe test_reroute_traffic, but I am just not sure how clear enough it sounds, since when we start routing, we are re-routing traffic, aren't we? :smile: